README.md

```diff
@@ -2,107 +2,8 @@
 
 [![Actions Status](https://github.com/exoscale/vault-plugin-secrets-exoscale/workflows/CI/badge.svg)](https://github.com/exoscale/vault-plugin-secrets-exoscale/actions?query=workflow%3ACI)
 
-This is a [backend plugin][vaultdocplugins] plugin to be used with HashiCorp [Vault](https://www.vaultproject.io/). This plugin generates Exoscale IAM API keys which can be restricted to specific operations according to predefined roles.
+This is a [HashiCorp Vault](https://www.vaultproject.io/) secrets engine that generates [Exoscale IAM API keys](https://community.exoscale.com/documentation/iam/) with specific [IAM Roles](https://community.exoscale.com/documentation/iam/iam-api-key-roles-policies/#iam-roles).
 
-This guide assumes you have already installed Vault and have a basic understanding of how Vault works. Otherwise, first read this guide on how to [get started with Vault][vaultdocintro].
+You can find the official documentation of this plugin on [the wiki](https://github.com/exoscale/vault-plugin-secrets-exoscale/wiki) of this repo.
```

Comment:
The doc is similar to the README for now, but the idea is to make it evolve into something more detailed and better organized (getting started, build from source, install, upgrade, manage IAMv3 roles etc.)